lib/features/scan_nfc/widgets/redirect_widget.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+
+class RedirectPopWidget extends StatefulWidget {
+  const RedirectPopWidget({super.key});
+
+  @override
+  State<RedirectPopWidget> createState() => _RedirectPopWidgetState();
+}
+
+class _RedirectPopWidgetState extends State<RedirectPopWidget> {
+  @override
+  void initState() {
+    context.pop();
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return const Center(
+      child: Text('USER SHOULD NOT SEE THIS SCRREN'),
```

Comment:
Even if the user shouldn't see this, we shouldn't put this text here ;) (it's a bit weird if a user sees this anyway)

Reply:
Put a button to return to splash which redirects to log in/profile selection/wallet screen.